foodfuncs.py

```
@@ -1,403 +1,305 @@
-
-    try:
-        conn = create_connection("recipes.db")
-        cur = conn.cursor()
-        pick = random.randint(1, rec_id)
-
-        recipe_pick = (cur.execute("SELECT recipe_name FROM recipes WHERE id = ?", (pick,),).fetchall())[0][0]
-        instructions_pick = (cur.execute("SELECT instructions FROM recipes WHERE id = ?", (pick,),).fetchall())[0][0]
-        ingredient_id_quantity = cur.execute("SELECT * FROM ingrecipe WHERE recipe_id = 4").fetchall()
-        
-        finaldict = {'name' : recipe_pick,
-                    'instructions' : instructions_pick,
-                    'ingredients' : {}
-        }
-        for x in ingredient_id_quantity:
-            ingredient_name = cur.execute("SELECT ingredient_name FROM ingredients WHERE id = ?",(x[1],),).fetchall()[0][0]
-            finaldict['ingredients'][ingredient_name] = x[2]
-        print(finaldict)
-        # use recipe id to get a list of ingredient ids from ingrecipe
-        # then use the ing ids to get ingredient names in loop with ing quantity
-    except sqlite3.Error as error:
-        print("Error: cannot connect or find the ingredient.", error)
-    finally:
-        if conn:
-            conn.close()
-
-
-    # iter = list(recipes.items())
-    # pick = random.choice(iter)
-
-    # while all(x in foodavail for x in pick[1])is not True:
-    #     pick = random.choice(iter)
-
-    # return pick 
-
-def create_connection(db_file):
-    """ create a database connection to the SQLite database
-        specified by db_file
-    :param db_file: database file
+def create_connection():
+    """ create a database connection to the SQLite database recipes.db
     :return: Connection object or None
     """
     conn = None
     try:
         # waits for max of 30 seconds before closing the db to account for large recipes
-        conn = sqlite3.connect(db_file, timeout=30.0)
+        conn = sqlite3.connect("recipes.db")
         return conn
     except Error as e:
         print(e)
```

I think you want to raise the error here not just print it